tests/unit/event_source/test_aws_cloudtrail.py

```diff
@@ -2,11 +2,12 @@
 from unittest.mock import patch
 
 import pytest
+from ansible_collections.ansible.eda.plugins.event_source.aws_cloudtrail import (
```

Comment:
It is fine to start the import from `plugins`. My local pytest will not even recognize `ansible_collections.ansible.eda`.

Reply:
it seems we do need a full path. Thank you.